src/Task/PhpStan.php

```diff
@@ -80,7 +85,7 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->addOptionalArgument('--autoload-file=%s', $config['autoload_file']);
         $arguments->addOptionalArgument('--configuration=%s', $config['configuration']);
         $arguments->addOptionalArgument('--memory-limit=%s', $config['memory_limit']);
-        $arguments->add(sprintf('--level=%u', $config['level']));
+        $arguments->addOptionalIntegerArgument('--level=%s', $config['level']);
```

Comment:
This will break on PHP 7.1 upgrade which will contain typehint `?int`. Can you create a `addOptionalMixedArgument()` instead?

Reply:
Sure. But without type hints both methods will look exactly the same. How should we deal with that?

Reply:
is something like this okay for you?
```php
# src/Collection/ProcessArgumentsCollection.php

public function addOptionalIntegerArgument(string $argument, $value)
{
    return $this->addOptionalMixedArgument($argument, $value);
}

public function addOptionalMixedArgument(string $argument, $value)
{
    if (null === $value) {
        return;
    }

    $this->add(sprintf($argument, $value));
}
```

This still works after adding the `?int` type hint

Reply:
good point! The solution you've provided is good for me. That allows us to have a type check for integers in the future.